docs/developers/AI.md

```diff
@@ -0,0 +1,34 @@
+< [Documentation](../Readme.md) /  AI Description - WIP
```

Comment:
```suggestion
< [Documentation](../Readme.md) /  AI Description
```

All our documentation is basically WIP :)

Reply:
Yes but this is really formatted copy paste from chat :)

Reply:
I'd rather remove it.
1) As above, all our docs are wip
2) Not sure if AI docs will get more updates soon. Meaning that this "WIP" tag will likely stay for quite some time.